runtime/convertValues_test.go

```diff
@@ -31,19 +31,21 @@ import (
 )
 
 var convertTests = []struct {
-	label    string
-	value    interpreter.Value
-	expected cadence.Value
+	label       string
+	value       interpreter.Value
+	expected    cadence.Value
+	skipReverse bool
 }{
 	{
 		label:    "Void",
 		value:    interpreter.VoidValue{},
 		expected: cadence.NewVoid(),
 	},
 	{
-		label:    "Nil",
-		value:    interpreter.NilValue{},
-		expected: cadence.NewOptional(nil),
+		label:       "Nil",
+		value:       interpreter.NilValue{},
+		expected:    cadence.NewOptional(nil),
+		skipReverse: true,
```

Comment:
Why is this test case skipped? A Cadence `nil` seems to be represented as `cadence.NewOptional(nil)`, which should convert to `interpreter.NilValue{}`

Reply:
I'm going to remove this test case in a follow-up PR. I liked your idea to introduce a `cadence.Nil` value, so I'l add that and then fix the implementation if that sounds good?

Reply:
Sounds good!